dockers/docker-dhcp-relay/Dockerfile.j2

```diff
@@ -1,26 +1,11 @@
-FROM docker-base
+FROM docker-config-engine
 
 ## Make apt-get non-interactive
 ENV DEBIAN_FRONTEND=noninteractive
 
 ## Update APT package lists
 RUN apt-get update
 
-## Dependencies for sonic-cfggen
-RUN apt-get -y install python-lxml python-jinja2 python-netaddr python-ipaddr python-yaml
-
-
-COPY \
```

Comment:
Should rename from Dockerfile.j2 to Dockerfile as well? SIMPLE_DOCKER don't use dockerfile templates